audbcards/core/dataset.py

```diff
@@ -361,6 +375,40 @@ def version(self) -> str:
         r"""Version of dataset."""
         return self._version
 
+    def _cached_properties(self):
+        """Get list of cached properties of the object."""
+        class_items = self.__class__.__dict__.items()
+        props = dict(
+            (k, getattr(self, k))
+            for k, v in class_items
+            if isinstance(v, functools.cached_property)
+        )
+        return props
+
+    def _load_backend(self) -> audbackend.Backend:
+        r"""Load backend containing dataset."""
+        backend = audbackend.access(
+            name=self.repository_object.backend,
+            host=self.repository_object.host,
+            repository=self.repository,
+        )
+        if isinstance(backend, audbackend.Artifactory):
+            backend._use_legacy_file_structure()  # pragma: nocover
```

Comment:
Why is the pragma necessary?

Reply:
You are right, this is not necessary as we request datasets from Artifactory. I guess this was just a copy and paste error.

I removed the `pragma` now.

Reply:
Then I cam resolving this.ü